scripts/qa_crowbarsetup.sh

```diff
@@ -449,6 +449,21 @@ function add_ha_repo()
     done
 }
 
+function add_ha12_repo()
+{
+    # TODO: add Updates-Test repo
+    local repo
+    for repo in SLE12-HA-{Pool,Updates}; do
+        if [ "$hacloud" == "2" -a "$repo" == "SLE12-HA-Updates-test" ] ; then
```

Comment:
The if clause will never hit. The repo name will never be the one you are testing.
So these lines can just be dropped.